en/developer-docs/docs/develop-components/deploy-an-application-with-buildpacks.md

```
@@ -187,3 +187,20 @@ e.g., <br>
 - java -jar target/sample.jar
 - node app.js
 - php -S 0.0.0.0:8000 index.php
+
+
+## Troubleshooting Go build issues
+
+If you encounter errors similar to:
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)